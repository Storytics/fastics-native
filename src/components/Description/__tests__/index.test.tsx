import * as React from "react";
import * as renderer from "react-test-renderer";
import Description from "..";

describe("Description component UI", () => {
  test("Should render properly", () => {
    const tree = renderer
      .create(<Description>Fastics Native Description</Description>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
