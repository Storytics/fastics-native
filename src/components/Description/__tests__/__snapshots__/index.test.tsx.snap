// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description component UI Should render properly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "grey",
        "fontSize": 14,
        "marginTop": 2,
      },
    ]
  }
>
  Fastics Native Description
</Text>
`;
